packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.ts

```diff
@@ -1354,6 +1354,9 @@ export function getDistanceBetweenPoints(point1: IPoint, point2: IPoint): number
 // @public
 export function getDocument(rootElement?: HTMLElement | null): Document | undefined;
 
+// @public
+export function getDocumentRTL(): boolean;

```

Comment:
this feels wrong.

Reply:
In implementation or API? Don't want to expose it?

Reply:
I'll remove this API addition.